### DISCUSSION TITLE
question: is it possible to limit an oidc group to access to certain templates?

### DISCUSSION THREAD

**Discussion by tuananh** [coder/coder#15678] [General] [answered]
is it possible to limit users of an oidc group to have access to only certain templates?

**Comment by matifali** [comment, marked as answer]
Yes, by making use of [groups](https://coder.com/docs/admin/users/groups-roles#groups) you can assign different groups different templates access. 
Pair this up with [role-sync](https://coder.com/docs/admin/users/idp-sync#role-sync-enterprise-premium) and you can achieve what you want. 
These are Premium features.

**Reply by tuananh** [reply]
Thank you. That's great to know

**Reply by matifali** [reply]
To add to it you can also explore our [`terraform-provider-coderd`](https://registry.terraform.io/providers/coder/coderd/latest/docs/resources/template) to manage ACLs via code.
